generators/entity/index.js

```diff
@@ -35,9 +34,16 @@ var fieldNamesUnderscored = ['id'];
 var fieldNameChoices = [], relNameChoices = []; // this variable will hold field and relationship names for question options during update
 var databaseType;
 var prodDatabaseType;
-const INTERPOLATE_REGEX = constants.INTERPOLATE_REGEX;
-const RESOURCE_DIR = constants.RESOURCE_DIR;
-const ANGULAR_DIR =  constants.ANGULAR_DIR;
+
+const constants = require('../generator-constants'),
+    INTERPOLATE_REGEX = constants.INTERPOLATE_REGEX,
+    CLIENT_MAIN_SRC_DIR = constants.CLIENT_MAIN_SRC_DIR,
+    CLIENT_TEST_SRC_DIR = constants.CLIENT_TEST_SRC_DIR,
+    ANGULAR_DIR =  CLIENT_MAIN_SRC_DIR + 'app/',
```

Comment:
Shouldn't this be in the constant file as well so that if we want to change the app folder it can be done in a single place?